packages/angular/cli/commands/add.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "AddCommandOptions",
+  "description": "Add support for a library to your project.",
+  "$longDescription": "",
+
+  "$scope": "in",
+  "$impl": "./add-impl#AddCommand",
```

Comment:
It seems we still have to maintain a bunch of ${command}.json files internally. I don't know how often will these json files change? Will it make sense for us to maintain a different set of json files?

Reply:
I want to unify schematics, architect and cli json schema files. There is already one code path but I want to make it more streamlined. This could also open up the way to custom commands down the road when we want to.